modules/spacetimedb-quickstart/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 use spacetimedb::{log, ReducerContext, Table};
 
-#[spacetimedb::table(name = person, public)]
+#[spacetimedb::table(name = person, public, index(name = age, btree(columns = [age])))]
```

Comment:
You could also do an index annotation on `age`. Except it might not have an index generated clientside. Hmm. I wonder if it would cause Rust module compilation errors to have both a column and unique index on a field.

Reply:
I think I did this because we don't yet have syntax for that in C# and I wasn't sure if an inline annotation in Rust would be completely equivalent to outer one in C#.

Might be better to keep them the same until we implement inline annotations in C# too.